### ISSUE TITLE
use yapf for auto-indentation

### ISSUE THREAD

**Issue by igalic** [iocage/libiocage#159]
rather than having a lot of text in CONTRIBUTING.md for exactly how to indent our code, we should use [yapf](https://github.com/google/yapf), now that it finally supports python 3.6!

**Comment by skarekrow** [comment]
I’ve been using it in iocage. It’s great!

**Comment by igalic** [comment]
@skarekrow what config are you using?
the one i use in #160 produced a giant diff that looks different form our style?
would you mind helping with that PR?

**Comment by gronke** [comment]
Seems to be a good candidate for a make command like `make beautiful` or `make the code look better`.

@skarekrow what were your learnings from implementing it for iocage?

**Comment by skarekrow** [comment]
I agree that it is a good candidate for that. As for learnings, not many. I let my editor do it with [ALE](https://github.com/w0rp/ale), instead of manually running it with any rules.
